paimon-common/src/main/java/org/apache/paimon/fileindex/FileIndexer.java

```diff
@@ -29,7 +30,7 @@ public interface FileIndexer {
 
     FileIndexWriter createWriter();
 
-    FileIndexReader createReader(byte[] serializedBytes);
+    FileIndexReader createReader(SeekablePositionedMappingInputStream inputStream);
```

Comment:
Does here need `SeekablePositionedMappingInputStream`? Can it just be `SeekableInputStream`?